src/elf_message/elf_message_field.c

```diff
@@ -0,0 +1,33 @@
+#include "global.h"
+#include "z64elf_message.h"
+
+ElfMessage gOverworldNaviMsgs[] = {
+/* 0000   0 */ ELF_MESSAGE_FLAG(CHECK, 0x0140, false, 0x05), /* eventChkInf[0] & 0x20 */
```

Comment:
Just a suggestion to keep things more concise, but we could shorten all these macros to `ELF_MSG_`